packages/api/src/utils/zod.ts

```diff
@@ -223,8 +223,8 @@ export const alertSchema = z
 // External API Alerts
```

Comment:
I think we need to update zod's `alertSchema` so the validator works properly with the new schema. Also one idea is to use zod to do the parsing to make sure the input string is in JSON stringified format (ref: https://github.com/colinhacks/zod/discussions/2215)

Reply:
@wrn14897 I'm actually not sure we need to change anything here for the time being now that I've rolled back the channel changes I made. channel/message format stays the same and I don't think the webhook schema changes have an effect here but let me know what you think.

Reply:
ah sorry, I was thinking about the webhook validator instead 🤦. And I realized we don't validate the webhook schema in the POST endpoint...

Reply:
:) all good - currently i'm adding naive validation to the new webhook form inputs, and im not overly zod familiar, but im guessing we could ramp that up with some much better validation as you suggested above later